tracer/src/Datadog.Trace/Sampling/DefaultSamplingRule.cs

```diff
@@ -5,16 +5,19 @@
 
 using System;
 using System.Collections.Generic;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.NLog.DirectSubmission.Formatting;
```

Comment:
Do we need this? Looks like it was auto-added by the IDE.

Reply:
🤔 , dunno how it has been added but indeed not needed. Thanks.
Fixed in 3d870399a080c9ad153a1f2579491abef18303d7